storage/src/main/java/org/apache/uniffle/storage/handler/impl/MemoryQuorumClientReadHandler.java

```diff
@@ -33,6 +33,7 @@ public class MemoryQuorumClientReadHandler extends AbstractClientReadHandler {
   private static final Logger LOG = LoggerFactory.getLogger(MemoryQuorumClientReadHandler.class);
   private long lastBlockId = Constants.INVALID_BLOCK_ID;
   private List<MemoryClientReadHandler> handlers = Lists.newLinkedList();
+  private boolean readAfterFirstRound;
```

Comment:
Why do we need this variable?

Reply:
The purpose is make sure at least one server can be read successfully in the first round. And i think we can remove it now.